Failure injector integration plugin implementation. (#16445)
elastic/logstash: qa/support/logstash-integration-failure_injector/README.md

# About the plugin

This plugin is for Logstash pipelines test purpose _only_.

# How to apply changes

> When you change the source code, make sure to build a gem file.

Author: ```suggestion
When you change the source code, remove existing `logstash-integration-failure_injector-{version}.gem` and  build a gem file with:
```
